src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,25 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.TaskList;
+
+public class ByeCommand extends Command {
+    private String commandMessage = "";
+    public ByeCommand() {
+    }
+
+    @Override
+    public String getCommandMessage() {
+        return commandMessage;
+    }
+    @Override
+    public void execute(TaskList taskList, Storage storage) {
+
+        commandMessage = "Byebye. See you again!";
+    }
+
+    @Override
+    public boolean isExitYet() {
```

Comment:
can naming convention be improved? such as just isExit()

Reply:
Okay! Will change. Thanks :)